Automattic/wp-calypso
# FSE: bump newspack-blocks version to 1.4.0

#### Changes proposed in this Pull Request

Bump newspack-blocks version to 1.4.0 to include the [latest changes as listed here](https://github.com/Automattic/newspack-blocks/releases/tag/v1.4.0).
#### Testing instructions



* Add Posts Carousel block to a page
* See if the 1.4.0 changes are applied e.g. the author shows up in the editor
